example/example.dart

```diff
@@ -0,0 +1,13 @@
+library touppercase.tests;
```

Comment:
Add copyright header? Remove library?

Reply:
Library is needed for reflection. Added copyright header.